csrankings-m.csv

```diff
@@ -131,6 +131,7 @@ Mahmuda Naznin,BUET,https://sites.google.com/site/mahmudanaznin,dJx58AQAAAAJ
 Mahmut T. Kandemir,Pennsylvania State University,http://www.cse.psu.edu/~mtk2,j67v24EAAAAJ
 Mahmut Taylan Kandemir,Pennsylvania State University,http://www.cse.psu.edu/~mtk2,j67v24EAAAAJ
 Mahsa Mirzargar,University of Miami,https://ccs.miami.edu/team_member/mahsa-mirzargar-phd,nm7Oe6sAAAAJ
+Mahsa Shoaran,EPFL,https://people.epfl.ch/mahsa.shoaran,9tu1zw4AAAAJ

```

Comment:
Not clear how meets inclusion criteria (able to solely advise a PhD student for a degree in CS).